### PR TITLE
[zh-cn] Update 'Roles and Teams'

### PR DESCRIPTION
**Pull request by jasonren0403**
### Motivation

Complete meta docs

### CODE REVIEW

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:141]:
```suggestion
杰出贡献者是那些对 MDN Web 文档做出了卓越贡献的人。他们的贡献形式包括：提出拉取请求以改进项目；在各种[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)或学习论坛上帮助社区成员；或就 GitHub 问题和拉取请求提供反馈。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:28]:
```suggestion
- 在[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)中参与社区讨论。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:188]:
```suggestion
- 监控所有[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:10]:
```suggestion
社区贡献对这个开源项目帮助巨大。贡献者可以通过他们在 MDN Web 文档上的工作展示他们的写作、技术和协作技能，以及与来自不同背景的人合作的能力。本节将介绍你在 MDN Web 文档项目志愿工作中可以担任的角色。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:24]:
```suggestion
如果你是新人，并且想成为一名贡献者，可以看看我们的[贡献指南](https://github.com/mdn/content/blob/main/CONTRIBUTING.md)和 [MDN GitHub 组织中的仓库列表](https://github.com/orgs/mdn/repositories)。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:33]: How about "参与"

```suggestion
- 参与社区活动。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:36]: I guess the tone of "需求" is relatively weak, should we use "要求"?

```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:41]: This link is broken, and it seems that the file was never created in the mdn/mdn-community repository. Hey @bsmth. Is there any plan to creating this, or should we state that this file is located in the root directory of each repo.

https://github.com/mdn/content/blob/27e8dec205645aa69d3171e7004191a6066debf3/files/en-us/mdn/community/roles_teams/index.md?plain=1#L48-L51

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:104]:
```suggestion
所有者拥有管理用户和 [GitHub 团队](https://github.com/orgs/mdn/teams)、维护 [MDN 组织](https://github.com/mdn)中各仓库的访问权限、维护仓库设置以及部署到生产环境的广泛权限。所有者受其他贡献者角色的所有要求约束。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:106]:
```suggestion
> **备注：** 所有者角色目前仅限于 Mozilla 员工。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:124]:
```suggestion
- 根据需要添加仓库（作为新项目或转移）。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:191]:
```suggestion
## 流程
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:207]: Cloud you also split those lines for the en-US version?

https://github.com/mdn/content/blob/27e8dec205645aa69d3171e7004191a6066debf3/files/en-us/mdn/community/roles_teams/index.md?plain=1#L242-L243

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:277]: Could we align the translation of `2FA`. There are two translations above that differ from this. I prefer "双因素身份验证"

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:64]:
```suggestion
- 在你的 GitHub 账户开启[双因素身份验证](https://docs.github.com/zh/authentication/securing-your-account-with-two-factor-authentication-2fa)。
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:131]:
```suggestion
| [**组织成员**](#组织成员) | - 在 GitHub 账户上启用双因素身份验证（2FA）<br> - 启用签名提交<br><br>至少一个：<br>- 解决至少两个议题<br> - 至少贡献了两个月<br>- 在一个项目领域作出活跃贡献            | 组织级别的访问权限                                     |
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:54]:
```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:85]:
```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:108]:
```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:157]:
```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (jasonren0403) [reply files/zh-cn/mdn/community/roles_teams/index.md:207]: PR raised

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:62]:
```suggestion
以下两个要求是必须满足的：
```

**Review by yin1999 — APPROVED** [review]
LGTM. Thanks @jasonren0403. Let's wait for the response from bsmth before merging :)

File: files/zh-cn/mdn/community/roles_teams/index.md
Comment (bsmth) [reply files/zh-cn/mdn/community/roles_teams/index.md:41]: >Is there any plan to creating this, or should we state that this file is located in the root directory of each repo.

There's no plan to create this, no: we're already linking to code of conduct (Community Participation Guidelines) so contribution guide should be per project `CONTRIBUTING.md`. I'm fine with saying 'check the `CONTRIBUTING.md` file in each repo for guidance' as you suggested 👍🏻

Comment (yin1999) [reply files/zh-cn/mdn/community/roles_teams/index.md:41]: @jasonren0403 could you also fix this ;)

Comment (jasonren0403) [reply files/zh-cn/mdn/community/roles_teams/index.md:41]: > @jasonren0403 could you also fix this ;)

Should we point to `translated-content` in localized content, and `content` in en-US content? 🤔 

```suggestion
- [MDN 的贡献准则](https://github.com/mdn/translated-content/blob/main/CONTRIBUTING.md)
```

Comment (yin1999) [reply files/zh-cn/mdn/community/roles_teams/index.md:41]: Maybe something like this:

```md
- Contribution guidelines (check the `CONTRIBUTING.md` file in each repo)
```

**Review by yin1999 — APPROVED** [review]
Thanks @jasonren0403 and @bsmth :)

**Comment by bsmth** [comment]
Thanks, both!

**Pull request merged**
